specification/src/main/asciidoc/platform/PlatformOverview.adoc

```diff
@@ -199,8 +199,10 @@ bean container.
 ==== Container Requirements
 
 This specification requires that containers
-provide a Java Compatible™ runtime environment, as defined by the Java
-Platform, Standard Edition, v8 specification (Java SE). The applet
+support execution in a Java™ runtime environment, as defined by the Java
+Platform, Standard Edition, v11 specification (Java SE 11). Containers may additionally choose
```

Comment:
Hmmm...  This makes it sound like we have defined Java SE 11 as the new minimum for Jakarta EE 9.1.  We allow the execution with Java SE 11.  And, we will also provide a certification test suite for Java SE 11.  But, our minimum is still Java SE 8.  Shouldn't we just leave this statement to say v8 (Java SE 8)?  And, the follow on statement says that you could use later versions as well...

Reply:
@kwsutter This may be a misunderstanding on my part. My impression was compatible implementations were required to support SE 11, and then could optionally support other SE versions. But I don't actually see that requirement for 11 in https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9.1ReleasePlan. Perhaps I was mixing up the requirement for at least one compatible impl on 11 in order to release 9.1 with a general requirement.

In any case, even if my understanding was correct, there's a good argument for not getting into that in this document, i.e. one similar to the one you make at https://github.com/eclipse-ee4j/jakartaee-platform/pull/302/files#r608951666. If vendors are required to certify on 11 in order to certify on others, that should be spelled out elsewhere. Where I don't know. :)

So I'll tweak this.

Reply:
I just pushed up a number of changes in response to your comments, but didn't get to this one yet.

Reply:
Fair enough.  A lot to digest so late in the evening... :-)  Jakarta EE 9.1 will *allow* for Java SE 11 certification, but it's not a requirement.  And, we definitely are not raising the minimum version of the Java SE runtime.  That is why we are still requiring all of the Class files to be compiled at the Java SE 8 class format version.  Compatible products for Jakarta EE 9.1 will be allowed to certify with Java SE 11, or Java SE 8, or both.

Reply:
I've updated this to just state that SE 8 or later is required.